requirements/base.txt

```diff
@@ -385,7 +385,7 @@ vine==5.1.0
     #   kombu
 wcwidth==0.2.13
     # via prompt-toolkit
-werkzeug==3.0.3
+werkzeug==3.0.6
```

Comment:
Do we have to change requirements/base.in too? Looks like it's doing >= 3.0.1 so if someone does a pip-compile, this can be downgraded again, right?